service-front/module/Application/src/Controller/Console/DynamoDbController.php

```diff
@@ -4,13 +4,10 @@
 
 use Aws\DynamoDb\Exception\DynamoDbException;
 use Aws\Sdk;
-use Opg\Lpa\Logger\LoggerTrait;

```

Comment:
noticed that this had been removed, but not replaced. having looked at the file , it has a bunch of echo statements. wondering if we need to do something here (could be a further TD ticket...)

Reply:
It's a console app, so rather than using the logger, it writes direct to stdout. Not sure if we need to do anything more than that as we rarely use this anyway (do we?).